setup.cfg

```diff
@@ -1,4 +1,2 @@
-;config file for the pep8 analysis plugin
-[pep8]
-max-line-length = 100
-exclude=tests
+[pycodestyle]
+max-line-length = 110
```

Comment:
If we are changing the max line length to 110, please update the contributing page to reflect this change as well.

Reply:
its to give some leeway, contributors should still aim for 100 but ive set it not to go off til over 110.
that gives contributers +- 10 characters before builds start failing

Reply:
reduced to +2.